### DISCUSSION TITLE
How to enable vulnerabilityAlerts for paths that are disabled

### DISCUSSION THREAD

**Discussion by batbattur** [renovatebot/renovate#22954] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.140.3

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Currently, I have updates disabled with `enabled: false` for some paths with the following config. How do I enable only the vulnerability updates for these paths?

```
module.exports = {
   extends: ['config:base'],
   ...
   packageRules: [
      {
         matchPaths: [
            'abc/**',
            'cde/**',
            'efg/**',
         ],
         enabled: false,
      },
   ],
};
```

### What you tried so far.

Tried adding `vulnerabilityAlerts: { enabled: true }` in the package rule but it didn't work.

```
module.exports = {
   extends: ['config:base'],
   ...
   packageRules: [
      {
         matchPaths: [
            'abc/**',
            'cde/**',
            'efg/**',
         ],
         enabled: false,
         vulnerabilityAlerts: {
             enabled: true
         }
      },
   ],
};
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate doesn't override _disabled_ dependencies because otherwise there would be no way for people to disable vulnerability updates that way (people who want to). Instead, you could try one of:
- Use dependencyDashboardApproval instead of disabling
- Use a schedule which will never be hit

**Reply by batbattur** [reply]
> Use a schedule which will never be hit

Is there a way to achieve this? I tried the following schedules, but none of them worked. I keep getting `WARN: Invalid schedule: Failed to parse` error.

1. `on 1st day of January in 2100`. Didn't work as well. Used the [later.js codepen](https://codepen.io/rationaltiger24/pen/ZExQEgK) to verify that it's valid. [Logs](https://github.com/batbattur/All/actions/runs/5468532033/jobs/9956301860#step:3:1096). 
2. `0 0 0 1 1 ? 1970`. Basically, tried to use a past cron schedule: https://stackoverflow.com/q/8324306.  [Logs](https://github.com/batbattur/All/actions/runs/5468734928/jobs/9956755536#step:3:953).
3. `never`. From the `tip` section in [here](https://docs.renovatebot.com/configuration-options/#schedule).  [Logs](https://github.com/batbattur/All/actions/runs/5468909027/jobs/9957170571#step:3:953).

<img width="1035" alt="Screen Shot 2023-07-05 at 2 20 20 PM" src="https://github.com/renovatebot/renovate/assets/58952979/9d36c4e8-0545-48f9-8674-ced14602a3f8">

**Reply by batbattur** [reply]
Found a way to do it. Set the schedule to run on February 31st (which will be never hit) 👍: `"schedule": ["* 0 31 2 *"]`
